test/commit-split.ts

```diff
@@ -29,12 +31,134 @@ describe('CommitSplit', () => {
   it('partitions commits based on path from root directory by default', async () => {
     const graphql = JSON.parse(
       readFileSync(
-        resolve(fixturesPath, 'commits-yoshi-php-monorepo.json'),
+        path.resolve(fixturesPath, 'commits-yoshi-php-monorepo.json'),
         'utf8'
       )
     );
     const commits: Commit[] = (await graphqlToCommits(github, graphql)).commits;
     const cs = new CommitSplit();
     snapshot(cs.split(commits));
   });
+
+  type ExpectedCommitSplit = Record<string, Commit[]>;
```

Comment:
I'd toss a comment above the two helpers, and above the actual setup for the assertions, pointing out:

1. what state each helper helps configure.
2. what methods we're testing with the assertion.